Master-OB-OpenAPI.json

```diff
@@ -18894,6 +18914,35 @@
           "description": ""
         }
       }
+    },
+    "ChecksumTypeItemType": {
+      "description": "",
+      "enums": {
+        "CRC": {
+          "label": "CRC",
+          "description": "Cyclical Redundancy Check - NIST SP 800-72"
+        },
+        "MD5": {
+          "label": "MD5",
+          "description": "Message Digest 5 - NIST SP 800-45 Version 2, NIST SP 800-101 Rev. 1"
```

Comment:
I think we should reference RFC 1321, April 1992.  Those NIST documents are guidelines on email and device security.